patches/react-native-audio-recorder-player+3.5.1.patch

```diff
@@ -0,0 +1,4025 @@
+diff --git a/node_modules/react-native-audio-recorder-player/android/build/.transforms/1074abf937aa2e6e3bf6d87bd7a32c8e/results.bin b/node_modules/react-native-audio-recorder-player/android/build/.transforms/1074abf937aa2e6e3bf6d87bd7a32c8e/results.bin
```

Comment:
I think it would be nice, if you can add the actual patch you wrote, not only the result the tool outputs, as it is impossible to understand what happens here, and also hard to check if the patch is needed after an update?

Reply:
i could add the the kotlin file i adjusted under patchfiles. and i removed the permission in the android manifest of the library. Gonna create a ticket to check if this has to be applied after next lib update

Reply:
https://issues.tuerantuer.org/browse/LUN-458